components/FeaturedCarousel.tsx

```diff
@@ -40,40 +39,73 @@ export const FeaturedCarousel: React.FC<FeaturedCarouselProps> = ({
 }) => {
   const [idx, setIdx] = useState(0);
   const currentItem = carouselData[idx];
-  const { title, category, description, image, profile, buttons } = currentItem;
+  const { title, category, description, image, organizations, buttons } = currentItem;
 
   const prev = () =>
     setIdx((i) => (i === 0 ? carouselData.length - 1 : i - 1));
   const next = () =>
     setIdx((i) => (i === carouselData.length - 1 ? 0 : i + 1));
 
+  // Handle multiple categories
+  const categories = Array.isArray(category) ? category : [category];
+
```

Comment:
delete this

Reply:
this is the line that is allowing for the flexibility. it's better to be stricter about types where possible. The less branching there is the cleaner the code will be